python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -75,11 +88,17 @@ def config_type(cls):
                 ),
             ),
             "secrets": Field(
-                Array(StringSource),
```

Comment:
We do lose backward compatibility with anybody who was setting their secret ARN from an env var - but that's probably fine because it's a bit of a chicken and egg story to figure out how you'd even set the env var in the first place.